### PR TITLE
Add WireGuard e2e test to travis

### PR DESCRIPTION
**Pull request by roytman**
Depends on https://github.com/submariner-io/submariner/pull/515

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,12 +18,15 @@ jobs:
   - env: CMD="make e2e"
          CLUSTERS_ARGS="--globalnet"
          DEPLOY_ARGS="${CLUSTERS_ARGS} --deploytool helm"
+  - env: CMD="make e2e"
+         DEPLOY_ARGS="--cable_driver wireguard"
```
Comment (mkolesnik) [inline .travis.yml:22]:
```suggestion
         DEPLOY_ARGS="--deploytool_submariner_args '--cable-driver wireguard'"
```

**Comment by mangelajo** [comment]
Very cool to see everything green, good job

**Comment by mangelajo** [comment]
Guys, do you mind if we hold back on adding this to travis, and we do it ASAP in GH actions?.

@mkolesnik tested it works pretty well, and we have the required capacity in GH actions (20 concurrent jobs).

**Comment by mangelajo** [comment]
@roytman  ^

**Comment by mangelajo** [comment]
As we plan to keep travis + GH actions in parallel for one cycle, travis would become a bottleneck.

**Comment by roytman** [comment]
> 
> 
> Guys, do you mind if we hold back on adding this to travis, and we do it ASAP in GH actions?.
> 
> @mkolesnik tested it works pretty well, and we have the required capacity in GH actions (20 concurrent jobs).

OK, no problem

**Comment by roytman** [comment]
waiting for GH actions

**Comment by mkolesnik** [comment]
The PR #551 is alive and kicking and even discovered issue #552 ;)

**Pull request closed without merging**